src/app/helptext/topbar.ts

```diff
@@ -40,8 +40,8 @@ export default {
 
     signupDialog: {
         title: T('Connect to TrueCommand Cloud'),
-        content: T('Do you want to connect to TrueCommand Cloud? \
-First-time connections require signing up with TrueCommand Cloud.'),
+        content: T('This allows your TrueNAS system to be monitored and administrated by\
+ TrueCommand. Click Signup to create a new TrueCommand Cloud instance, or Connect to join an existing instance.'),
```

Comment:
Removed unnecesary comma and added bold to specific button labels. Text is fine otherwise!
```suggestion
 TrueCommand. Click <b>Signup</b> to create a new TrueCommand Cloud instance or <b>Connect</b> to join an existing instance.'),
```